dts/bindings/mtd/jedec,spi-nor.yaml

```diff
@@ -1,26 +1,121 @@
-# Copyright (c) 2018 Peter Bigot Consulting, LLC
-# Copyright (c) 2019 Nordic Semiconductor ASA
+# Copyright (c) 2021, STMicroelectronics
 # SPDX-License-Identifier: Apache-2.0
 
 description: |
-  Properties supporting Zephyr spi-nor flash driver (over the Zephyr SPI
-  API) control of serial flash memories using the standard M25P80-based
-  command set.
+    STM32 OSPI Flash controller supporting the JEDEC CFI interface
```

Comment:
You're taking a generic binding to make it STM32 specific, that is for sure a "no, never".